numpy/core/src/common/simd/vec/conversion.h

```diff
@@ -170,7 +170,8 @@ npyv_pack_b8_b64(npyv_b64 a, npyv_b64 b, npyv_b64 c, npyv_b64 d,
     #ifdef NPY_HAVE_VXE2
         return vec_signed(a);
     #elif defined(NPY_HAVE_VXE)
-        return vec_packs(vec_signed(npyv_doublee(a)), vec_signed(npyv_doublee(vec_mergel(a, a))));
+        return vec_packs(vec_signed(__builtin_s390_vflls(vec_mergeh(a,a))),
+            vec_signed(__builtin_s390_vflls(vec_mergel(a, a))));
```

Comment:
I just realized that npyv_doublee is simply a
    #define npyv_doublee __builtin_s390_vflls
in utils.h So it would have been fine to continue using npyv_doublee instead of __builtin_s390_vflls

Reply:
Thanks @Andreas-Krebbel ! I have updated in latest commit.